ftw/upgrade/tests/test_upgrade_step_intids.py

```diff
@@ -0,0 +1,7 @@
+from ftw.upgrade.testing import INTID_UPGRADE_FUNCTIONAL_TESTING
+from ftw.upgrade.tests.test_upgrade_step import TestUpgradeStep
+
+
+class TestUpgradeStepIntids(TestUpgradeStep):
```

Comment:
Mhh yeah - this just re-runs all tests from `TestUpgradeStep` with a different layer to make sure that including `plone.app.intid` and thus running with the fix from this PR enabled does not break anything.
I could not figure out a way to test what is fixed with this PR since it is lowlevel stuff, see comment on PR aboce. But maybe you've got an idea?